app/models/Country.scala

```diff
@@ -230,8 +229,8 @@ object Country {
       .find(_.code == countryCode)
       .getOrElse(Country(countryCode, "Unknown"))
 
-  def selectItems(implicit messages: Messages): Seq[SelectItem] =
-    SelectItem(value = None, text = messages("countrySelect.label")) +:
+  def selectItems: Seq[SelectItem] =
+    SelectItem(value = None, text = "") +:
```

Comment:
this allows the field to be empty, without it, the autocomplete will pick the first country in the list

Reply:
As discussed, please revert, thanks.